src/Objective/Objective.ts

```diff
@@ -6,8 +7,18 @@ export class Objective {
     public Area: string[];
     public Target: Date;
     public Progress: number;
-    public KRs: string[];
+    public KRs: KR[];
     public WIT: string[];
-    public Status: string;
+    public Status: StatusType;
     public Comments: string[];
+    public TimeFrame: string;
+}
+
+export class KR {
+    public Id: string;
+    public ObjectiveId: string;
```

Comment:
Do we need this if the KR is part of the Objective document?

Reply:
I will remove it.